### DISCUSSION TITLE
I have Namecheap's custom email service enabled

### DISCUSSION THREAD

**Discussion by mahfuz-ta26** [emailjs-com/help#86] [Q&A]
Is it possible to implement the EmailJS mechanism with Namecheap email?

**Comment by xr0master** [comment]
I am sure, you can use the SMTP service to connect the Namecheap to the EmailJS

**Reply by mahfuz-ta26** [reply]
Okey, thank you.

Facing new problem. When I sent an email the response I find is that my templateId is invalid even though I copied it from the website.

This is how I am sending this:
        emailjs.send(serviceId,templateId,templateParams,publicKey)
            .then((response) => {
                console.log("Email sent successfuly: ",response)
            })
            .catch((err) => {
                console.log(`Error sending email: `,err)
            })
What could be the possible reason? If you can give me some hint it would be great. Thanks.

**Reply by xr0master** [reply]
Sorry, no clue. Try opening the Network panel and check what is sent in the request.
